src/main/java/com/google/sps/servlets/FindLocationServlet.java

```diff
@@ -19,7 +27,25 @@
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
     
-    //TODO - implement the rest of this function (@Kaleab)
+    String correlator = request.getParameter("correlator");
```

Comment:
Make const, for any being reused in multiple files, you can create a constants file that exports all the consts you will need

Reply:
Done!